foundation-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -266,14 +266,14 @@ The result for the `getUrlVariables` request:
 ```
 
 {% hint style="warning" %}
-The Analytics Custom VID is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/987092ee0fd2699ae5898755cd72255802628107/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+The Analytics Custom VID is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](../../../using-mobile-extensions/adobe-analytics/analytics-api-reference.md#get-the-custom-visitor-identifier).
```

Comment:
can you check if this goes to the correct link?

Reply:
The relative link to the page is correct, but the fragment is wrong. It should be `#getvisitoridentifier`. I'll fix it.